spinalcordtoolbox/scripts/sct_image.py

```diff
@@ -91,6 +92,15 @@ def get_parser():
         choices = ('uint8','int16','int32','float32','complex64','float64','int8','uint16','uint32','int64','uint64'))
 
     header = parser.add_argument_group('HEADER OPERATIONS')
+    header.add_argument(
+        "-header",
+        choices=HEADER_FORMATS,
+        # 'const' and 'nargs' used because of https://stackoverflow.com/q/40324356
+        const='sct',
+        nargs='?',
+        help="Print the header using the given output format. Choose 'nibabel' or 'fslhd' if you already have header "
+             "information from those programs and you want to compute the diff between two headers."
```

Comment:
I would remove the blurb about the diff (it will confuse our users). Also i changed the usage to make it clearer that adding the format is optional (our users don't understand the POSIX "[...]" semantic in usage).
```suggestion
        help="Print the header of a NIfTI file. You can select the output format of the header: 'sct' (default), 'nibabel' or 'fslhd'."
```